src/eplformulaire.cls

```diff
@@ -12,8 +12,8 @@
 \LoadClass{../../../eplbase}
 
 \IfLanguageName{english}{
-\newcommand{\Epltype}{Formulaire}
-\newcommand{\epltypesingular}{formulaire} % si quelqu'un connait la vraie traduction en anglais
+\newcommand{\Epltype}{Formulary}
+\newcommand{\epltypesingular}{formulary} % si quelqu'un connait la vraie traduction en anglais
```

Comment:
_I do not think it means what you think it means_ : "formulary" semble (d'après les dictionnaires que j'ai consulté) plutôt signifier un recueil de formules, dans le sens formules religieuses, de politesse, de pharmacie, et pas tellement une liste de formules et d'identités mathématiques. Peut-être qu'on pourrait en profiter pour utiliser une meilleure formulation comme "formulae list", "list of formulae" ou autre.

Reply:
En effet, certains dicos contredisent cette traduction mais je me suis inspiré de https://english.stackexchange.com/questions/105524/collection-of-mathematical-formulas, et "Handbook" me paraissait un peu étrange dans le cas d'un formulaire de quelques pages seulement.

Reply:
Handsheet alors ? Après, vu que les synthèses vont être utilisées par des gens majoritairement francophones, je pense que ce n'est pas un si grand problème que ça.

Reply:
Ou Cheatsheet ?

Reply:
J'aime bien "Cheatsheet"...

Reply:
Du coup je dois changer ou je laisse tel quel?

Reply:
Tu peux faire le changement dans un autre PR